src/hooks.ts

```diff
@@ -18,30 +18,20 @@ export const useCourseById = (
 export const useLoadFullData = (
```

Comment:
It's just a method, not hook. Avoid naming it starting from "use". It will cause a lot of confusion.